linuxNew/jdk/redhat/src/main/packaging/Dockerfile

```diff
@@ -4,14 +4,14 @@ RUN dnf update -y && dnf install -y rpmdevtools rpmlint gnupg2 wget tar dpkg fin
 
 RUN wget --progress=dot:mega -O /usr/bin/gosu https://github.com/tianon/gosu/releases/download/1.12/gosu-"$(dpkg --print-architecture | awk -F- '{ print $NF }')"
 RUN wget --progress=dot:mega -O /tmp/gosu.asc https://github.com/tianon/gosu/releases/download/1.12/gosu-"$(dpkg --print-architecture | awk -F- '{ print $NF }')".asc
-RUN gpg --batch --keyserver hkps://keys.openpgp.org --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4
+RUN gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4
 RUN gpg --batch --verify /tmp/gosu.asc /usr/bin/gosu
 RUN chmod +x /usr/bin/gosu
 RUN rm -f /tmp/gosu.asc
 
 RUN wget --progress=dot:mega -O /usr/bin/tini https://github.com/krallin/tini/releases/download/v0.19.0/tini-"$(dpkg --print-architecture | awk -F- '{ print $NF }')"
 RUN wget --progress=dot:mega -O /tmp/tini.asc https://github.com/krallin/tini/releases/download/v0.19.0/tini-"$(dpkg --print-architecture | awk -F- '{ print $NF }')".asc
-RUN gpg --batch --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 595E85A6B1B4779EA4DAAEC70B588DFF0527A9B7
+RUN gpg --batch --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 595E85A6B1B4779EA4DAAEC70B588DFF0527A9B7
```

Comment:
You have remove _--batch_ on the previous gpg line but not on this one.
Is it intentional ?

Reply:
Not intentional - I'll do some reading again and make it consistent, thank you!